### ISSUE TITLE
Gray links are hard to see and don’t pass accessibility requirements

### ISSUE THREAD

**Issue by glorialiou** [seattle-uat/civiform#1903]
Need more information - if just a color change, should be very low eng effort.

**Comment by rahaghas** [comment]
Change links from grey to action color as they are actionable elements and not disabled (grey can often be confused for disabled). This should also solve the accessibility issue.

**Comment by clouser2** [comment]
@rahaghas - mind clarifying what "action color is in this case?
@glorialiou - what links are specifically being talked about here? The top-level nav or other links? It may be somewhat difficult to audit all of the links in the UI that may fall into this case

**Comment by rahaghas** [comment]
Yeah so I'm not sure if it depends on the entity at this point, but it's whatever color is currently used as the background-color for filled buttons on the site

**Comment by glorialiou** [comment]
This is the context: https://docs.google.com/presentation/d/1goxiepPXPrW3wK8wUSNoCpcf7iXIohXfj96iTYiQxo0/edit#slide=id.gfa7bf315fa_0_1732

Adding screenshot for folks who don't have access to the doc:
<img width="1572" alt="Screen Shot 2022-05-23 at 12 33 48 PM" src="https://user-images.githubusercontent.com/11811064/169892985-36e78c4f-0072-4afc-affd-9c00e5979d1d.png">

**Comment by clouser2** [comment]
Thanks for the pointers. For static content displayed on the application, we run the entered text through a preprocessing helper to auto-detect links ([code](https://github.com/seattle-uat/civiform/blob/7dcb44b98660a18cf60c2adc9112a8c83860fb98/server/app/views/questiontypes/StaticContentQuestionRenderer.java#L35)). The underlying helper applies a 75% opacity to the rendered text ([code](https://github.com/seattle-uat/civiform/blob/7dcb44b98660a18cf60c2adc9112a8c83860fb98/server/app/views/components/TextFormatter.java#L81)).

Oddly, the links are still appearing as rendered gray. I can look into fixing it in this case to render with the "standard" link styling. I'll also audit our other usages of `TextFormatter` and see if we should just apply the styling generally.
